packages/core-typings/src/IWorkspaceCredentials.ts

```diff
@@ -0,0 +1,8 @@
+import type { IRocketChatRecord } from './IRocketChatRecord';
+
+export interface IWorkspaceCredentials extends IRocketChatRecord {
+	_id: string;
+	scopes: string[];
```

Comment:
This may be a dumb question, but do we really need scopes to be an array? :thinking: I see we're always setting AND searching it as an array with a single element

Reply:
Leaving it as an array is more of future-proofing than anything, it is not necessary nowadays, I think it was a suggestion by @geekgonecrazy

Reply:
I agree. Lists are almost always better.